core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -1430,6 +1430,47 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  // Send Change Notification for Workspace require WRITE access. Accept if OWNER or WRITE; Reject if READ or NO ACCESS
```

Comment:
I would like to keep all notification tests in the same place. Please move these to org.broadinstitute.dsde.rawls.webservice.NotificationSpec.